CoFrancePlugIn.cpp

```diff
@@ -517,6 +520,57 @@ void CoFrancePlugIn::OnGetTagItem(CFlightPlan FlightPlan, CRadarTarget RadarTarg
         string Stand = FlightPlan.GetControllerAssignedData().GetFlightStripAnnotation(CoFranceTags::ANNOTATION_STAND);
         strcpy_s(sItemString, 16, Stand.c_str());
     }
+
+    if (ItemCode == CoFranceTags::COLORED_RWY) {
+        string dep_rwy = "";
+        if (FlightPlan.IsValid()) {
+            if (strlen(FlightPlan.GetFlightPlanData().GetDepartureRwy()) > 0) {
```

Comment:
Would suggest assigning GetDepartureRwy to a variable and re-use to avoid ES api calls + readability